eventDisplay/src/EventDisplay.cc

```diff
@@ -314,22 +314,12 @@ void EventDisplay::drawEvent(unsigned int id, bool resetCam) {
 
       std::unique_ptr<AbsKalmanFitter> fitter;
       switch (fitterId_) {
-        case SimpleKalman:
```

Comment:
squareRootFormalism in its current implementation will be eradicated. According to Tobi it was never worth it. If we want to re-implement it, we can do so in a nicer way after the refactoring. Removing it like this makes refactoring more easy, because we have less legacy code to carry on.